### DISCUSSION TITLE
How to disable Hotkeys in SearXNG?

### DISCUSSION THREAD

**Discussion by kent57** [searxng/searxng#3137] [Q&A] [answered]
Is there a simple way to completely disable Hotkeys in SearXNG?

**Comment by return42** [comment, marked as answer]
No, at least the default keybindings are used ..

https://github.com/searxng/searxng/blob/047c7a7cf6d5ffc6df062ff4b9e6b71126dee063/searx/static/themes/simple/src/js/main/keyboard.js#L180

Any reason why you want to disable completely?

**Reply by kent57** [reply]
Thank you very much for your reply 

To be honest, it's because I want to change the phrase "How to navigate SearXNG with hotkeys" to something else (replace SearXNG for example).  But I don't know how.  I changed the HTML text in /searxng-src/searx/static/themes/simple/src/js/main/keyboard.js but that didn't work.
So I guess a better question is how do I change the HelpConent text? I cannot find that code  anywhere in the code - But I mave have missed it?

**Reply by kent57** [reply]
In hind-sight, I never wanted to disable the hotkeys.  I simply did not want the modal help screen to pop up.  My ultimate solution was to change the .css file that handled that popup and changed the display type to none.

**Comment by impaktor** [comment]
I'm also _very_ interested in disabling hotkeys, as I really hate it when fancy pages try to seal my keys. Naturally, I'm already using a browser that is keyboard driven (a common practice by tech-savvy I would expect), thus I rely on my existing key-map to follow or copy links (on all sites), but sadly some keys collide with SearXNG's key map, as it assumes I don't already have a keyboard driven browser. 

E.g. if I want to copy a link URL, I hit 'c', and all links will be indexed for me to then reference by number, or matching (narrow to) search phrase, but 'c'-key is intercepted by SearXNG. (gif demonstrating of how narrow to search works [here](https://github.com/atlas-engineer/nyxt/issues/2972))

Anyway, this project seems very interesting to me, and would be most excellent if there was a way to disable hotkeys. 

Duckduckgo, has a html only version of their page, that is lovely to navigate for me (although the default one works well also, at the present):
https://html.duckduckgo.com/html/
